react/components/SelectionListOrderBy.js

```diff
@@ -45,9 +44,9 @@ const SelectionListOrderBy = ({
     })
   }
 
-  const getOptionTitle = option => {
+  const getOptionTitle = useCallback((option) => {
```

Comment:
This works ? 🤔Shouldn't be something like `const getOptionTitle = useCallback(() => find(propEq('value', option), options).label, [option])`  ?

Reply:
@lucasecdb, what do u think?

Reply:
`option` is actually a parameter, we only need to add `options` to the deps array

Reply:
you can see the usage [here](https://github.com/vtex-apps/search-result/pull/116/files#diff-01b05f9607839211d3b9f685410dc260R75)